h2o-genmodel/src/main/java/hex/genmodel/MojoPipelineWriter.java

```diff
@@ -1,6 +1,7 @@
 package hex.genmodel;
 
 import hex.ModelCategory;
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
```

Comment:
definitely not the right exception

Reply:
yeah, it is true. Thanks! This mistake came from my Scala skills as there is NotImplementedError that is being used in such cases. Probably what I need here is java's alternative `UnsupportedOperationException`